classes/01class/exercises/c01-git02/mendoncaj/README.md

```diff
@@ -0,0 +1,28 @@
+# c01-git02
+
+## Answers
+
+1. Is it possible to see the inner commit of squashed commit with `git rebase`?
+
+```
+Yes, first find the hash commit[s]:
+
+$ git reflog <branch>
+
+note.: If the branch no longer exists, try: $ git reflog show 
+
+then if you need more details inspect the commit with
+
+$ git log -p bd8db00
+```
```

Comment:
What if you pull a squashed commit from GitHub, will you still be able to see those commits int the log or reflog ?